kmk/modules/layers.py

```diff
@@ -143,3 +165,41 @@ def _print_debug(self, keyboard):
         # debug(f'__getitem__ {key}')
         if debug.enabled:
             debug(f'active_layers={keyboard.active_layers}')
+
+    def get_combo_layer(self, keyboard):
+
+        acvlyrs = []
+
+        # If the active keyboard.active_layers value is greater than 0, then add it to the acvlyrs list
+        if len(keyboard.active_layers) > 0:
+            for i in keyboard.active_layers:
```

Comment:
similare to the above: give it a proper, recognisable name (unless there's a reasonably common convention for a abreviation).
```suggestion
            for layer in keyboard.active_layers:
```

Reply:
Fixed